### PR TITLE
docs: fix send rate usage in api.md

### PR DESCRIPTION
**Pull request by trainchou**
fix issue #2014

### CODE REVIEW

File: Documentation/2.0/api.md [hunk 1]
```diff
@@ -919,7 +919,7 @@ Each node keeps a number of internal statistics:
 - `recvBandwidthRate`: number of bytes per second this node is receiving (follower only)
 - `recvPkgRate`: number of requests per second this node is receiving (follower only)
 - `sendAppendRequestCnt`: number of requests that this node has sent
-- `sendBandwidthRate`: number of bytes per second this node is receiving (leader only). This value is undefined on single member clusters.
+- `sendBandwidthRate`: number of bytes per second this node has sent (leader only). This value is undefined on single member clusters.
 - `sendPkgRate`: number of requests per second this node is receiving (leader only). This value is undefined on single member clusters.
```
Comment (yichengq) [inline Documentation/2.0/api.md]: s/is receiving/is sending/

File: Documentation/2.0/api.md [hunk 2]
```diff
@@ -919,7 +919,7 @@ Each node keeps a number of internal statistics:
 - `recvBandwidthRate`: number of bytes per second this node is receiving (follower only)
 - `recvPkgRate`: number of requests per second this node is receiving (follower only)
 - `sendAppendRequestCnt`: number of requests that this node has sent
-- `sendBandwidthRate`: number of bytes per second this node is receiving (leader only). This value is undefined on single member clusters.
+- `sendBandwidthRate`: number of bytes per second this node has sent (leader only). This value is undefined on single member clusters.
```
Comment (yichengq) [inline Documentation/2.0/api.md]: s/has sent/is sending/

**Comment by yichengq** [comment]
Could you update it and rename the commit title to `docs: fix send rate usage in api.md`?

**Comment by xiang90** [comment]
@trainchou This patch looks good to me. But can you please squash the 3 commits to one, so I can merge it? Thanks!

**Comment by trainchou** [comment]
@xiang90 Yes, of course! Thanks for your suggestion!

**Comment by xiang90** [comment]
lgtm

**Pull request merged**